sklearn/utils/validation.py

```diff
@@ -345,7 +352,8 @@ def _ensure_no_complex_data(array):
 def check_array(array, accept_sparse=False, dtype="numeric", order=None,
                 copy=False, force_all_finite=True, ensure_2d=True,
                 allow_nd=False, ensure_min_samples=1, ensure_min_features=1,
-                warn_on_dtype=False, estimator=None):
+                warn_on_dtype=False, estimator=None,
+                accept_large_sparse=False):
```

Comment:
I am not sure this should default to false, as that may break current behaviour in external uses of check_array

Reply:
I know, this changes a lot of lines in the PR. Sorryi did not realise it sooner